source/deprecations/index.md

```diff
@@ -99,14 +99,19 @@ var App.ListView = Ember.View.extend(ListView);
 A more advanced solution is to use an initializer to register the plugin's
 views on the the application:
 
-```JavaScript
+```app/initializers/list-view.js
 // usage is {{view "list"}}
-Ember.Application.initializer({
-  name: 'list-view',
+
+export function initialize(container, application) {
   initialize: function(container, application) {
```

Comment:
This new version seems incorrect. it has both the old property and the new function.